src/services/submissionService.ts

```diff
@@ -158,6 +168,16 @@ export const updateSubmission = async (id: string, updatedFields: Partial<dbSche
     }
 }
 
+const validateSubmissionInputFields = (input: gqlType.SubmissionInput) => {
+    if (
+        (input.googleMapsUrl === null || input.googleMapsUrl === undefined || !input.googleMapsUrl.trim()) ||
```

Comment:
```suggestion
        (input.googleMapsUrl || !input.googleMapsUrl.trim()) ||
```

didn't see this sooner. you can simplify a null and undefined check with a basic truthy check. (truthy checks for both and empty strings, but you need to be careful if things like 0 are valid options.